src/transformers/models/cpmant/modeling_cpmant.py

```diff
@@ -395,7 +395,7 @@ def forward(
             if output_attentions:
                 all_self_attns += (attn_weights,)
             if current_key_value is not None:
-                current_key_values.append(current_key_value)
+                current_key_values = current_key_values + (current_key_value,)
```

Comment:
the convention is to use tuple instead of list, and I am a man following the rule.

Reply:
🫡